### DISCUSSION TITLE
Inserting unicode characters

### DISCUSSION THREAD

**Discussion by Sikarjan** [pqrs-org/KE-complex_modifications#1113] [Ideas]
Hi,

I am wandering if it would be possible to insert unicode characters with karabiner? For example remapping option+c to U+23C.
Specifically I am looking for a way to insert a hair space, U+200A, with option + space.

Thanks

**Comment by MuhammedZakir** [comment]
See https://github.com/pqrs-org/KE-complex_modifications/issues/697.

**Reply by Sikarjan** [reply]
Hmm, looks rather difficult. I am not a programmer but I will take a second look. I hoped for copying the right char into the "to" section😅

**Reply by MuhammedZakir** [reply]
It only "looks" difficult. If you have any question/doubt, feel free to ask them.
